src/fs/dvfs/dvfs_util.c

```diff
@@ -123,11 +124,15 @@ int dvfs_destroy_inode(struct inode *inode) {
 
 	if (inode->i_sb && inode->i_sb->sb_ops &&
 	    inode->i_sb->sb_ops->destroy_inode)
-		return inode->i_sb->sb_ops->destroy_inode(inode);
-	else
-		return dvfs_default_destroy_inode(inode);
+		inode->i_sb->sb_ops->destroy_inode(inode);
```

Comment:
dvfs_alloc_inode still use fs-specific allocate as complementary, which is inconsistent with dvfs_destroy_inode